java gr 8/homework/07homework7_obj_containers/src/main/java/ev/purchase/program/domain/GovernmentFinacedProgram.java

```diff
@@ -0,0 +1,38 @@
+package ev.purchase.program.domain;
+
+import javafx.beans.binding.FloatExpression;
+import javafx.beans.value.ObservableNumberValue;
+
+/**
+ * Created by Narci on 8/26/2017
+ */
+
+public class GovernmentFinacedProgram {
+
+    float fixedBudget = 10000000;
+    float greenBonus = 10000;
+
+
+    public GovernmentFinacedProgram ( ) {
+    }
+
+    public float getFixedBudget ( ) {
+        return this.fixedBudget;
+    }
+
+
+    public float getGreenBonus ( ) {
+
+        return this.greenBonus;
+    }
+
+    public int getGreenBonus ( GreenBonus bonus ) {
+        FloatExpression history = null;
+        history.add ( (ObservableNumberValue) bonus );
```

Comment:
What are you doing with these 2 lines? What are you using FLoatExpression and ObservableNumberValue for?